unit-tests/tests/33ignoreip.t

```diff
@@ -217,3 +217,36 @@ location /RequestDenied {
 --- request
 GET /foobar
 --- error_code: 200
+
+=== TEST 1.8: IgnoreIP request inheritance
+--- user_files
+>>> foobar
+foobar text
+--- main_config
+load_module $TEST_NGINX_NAXSI_MODULE_SO;
+--- http_config
+include $TEST_NGINX_NAXSI_RULES;
+--- config
+location / {
+     SecRulesEnabled;
+     IgnoreIP  "127.0.0.1";
+     DeniedUrl "/RequestDenied";
+     CheckRule "$SQL >= 8" BLOCK;
+     CheckRule "$RFI >= 8" BLOCK;
+     CheckRule "$TRAVERSAL >= 4" BLOCK;
+     CheckRule "$XSS >= 8" BLOCK;
+     root $TEST_NGINX_SERVROOT/html/;
+     index index.html index.htm;
+
+     location /foobar {
+          BasicRule wl:10;
+     }
+}
+location /RequestDenied {
+     return 412;
+}
+--- request
+GET /foobar?a=update/table
+--- curl
+--- curl_options: --interface 127.0.0.1
```

Comment:
this might be an issue for windows, since it is not supported.

Reply:
Any idea what other way you can make a request from a specific address?

In the description of Test::Nginx::Socket I did not find anything more suitable :sob:

Reply:
not really. but this is something i can patch.

Reply:
maybe not install new tests at all?

Reply:
nono. we need them. give me some times to patch this in the python code

Reply:
@lubomudr does this issue block your usage of naxsi?

Reply:
No it's not urgent

Reply:
I'm trying to make a patch for handling curl in python tests
But I hardly know him
And I can't test on Windows